src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -2068,7 +2065,7 @@ static void Main(string[] args)
         System.Console.WriteLine({|Rename:NewMethod|}());
     }
 
-    private static Index NewMethod()
+    private static System.Index NewMethod()
```

Comment:
> do we not want to add the using System here? 

So *technically* we do want to add `using System;` here and the ImportAdder *tries* to do just that.  However, this then triggers the conflict check and it feels like it would be unsafe to do that.  That's because it sees references to symbols in teh code that look like tehy could potentially collide if we added `using System;`.  i.e. lots of the refererence *inside* the `namespace System { ... }` block in the test code look like they could collide.  So we don't add it.

Reply:
If we updated the test to put the System.Index code into a separate source file, would that be a better test of reality? I'm not sure how easy that is to update the helpers for...